inc/class-core-sitemaps-renderer.php

```diff
@@ -75,5 +106,12 @@ public function render_sitemap( $url_list ) {
 		// All output is escaped within the addChild method calls.
 		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		echo $urlset->asXML();
+
+		/**
```

Comment:
Same as above. You want to put the filter inside the function that returns the stylesheet location rather than here. This function already echoes the created XML so it doesn't really return anything that would be useful to filter.